packages/wasm-dpp/test/unit/dataContract/DataContract.spec.js

```diff
@@ -194,13 +194,11 @@ describe('DataContract', () => {
 
   describe('#getDocumentSchema', () => {
     it('should throw error if Document is not defined', () => {
-      let error;
       try {
         dataContract.getDocumentSchema('undefinedObject');
```

Comment:
There needs to be a fail here, otherwise, the test will pass if no error is thrown. The old way did account for that actually